jest.config.js

```diff
@@ -1,10 +1,36 @@
 module.exports = {
-  testMatch: ['**/+(*.)+(spec|test).+(ts|js)?(x)'],
-  transform: {
-    '^.+\\.(ts|js|html)$': 'ts-jest',
-  },
-  resolver: '../../scripts/patched-jest-resolver.js',
-  moduleFileExtensions: ['ts', 'js', 'html'],
-  coverageReporters: ['html'],
-  maxWorkers: 2,
+  projects: [
+    '<rootDir>/packages/tao',
+    '<rootDir>/packages/workspace',
+    '<rootDir>/packages/web',
+    '<rootDir>/packages/cypress',
+    '<rootDir>/packages/jest',
+    '<rootDir>/packages/storybook',
+    '<rootDir>/packages/react',
+    '<rootDir>/packages/nx-plugin',
+    '<rootDir>/packages/node',
+    '<rootDir>/packages/next',
+    '<rootDir>/packages/nest',
+    '<rootDir>/packages/linter',
+    '<rootDir>/packages/express',
+    '<rootDir>/packages/eslint-plugin-nx',
+    '<rootDir>/packages/create-nx-workspace',
+    '<rootDir>/packages/create-nx-plugin',
+    '<rootDir>/packages/cli',
+    '<rootDir>/packages/bazel',
+    '<rootDir>/packages/angular',
+    '<rootDir>/e2e/cli',
+    '<rootDir>/e2e/angular',
+    '<rootDir>/e2e/bazel',
+    '<rootDir>/e2e/cypress',
+    '<rootDir>/e2e/jest',
+    '<rootDir>/e2e/linter',
+    '<rootDir>/e2e/next',
+    '<rootDir>/e2e/node',
+    '<rootDir>/e2e/react',
+    '<rootDir>/e2e/storybook',
+    '<rootDir>/e2e/web',
+    '<rootDir>/e2e/workspace',
+    '<rootDir>/e2e/nx-plugin',
```

Comment:
We might want to remove these from the root jest config. If someone runs `jest`, they're going to have a bad time because of these e2es 😅

Reply:
Good idea :)